src/legacy/core_plugins/kibana/public/home/tutorial_resources/logos/stan.svg

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Hi, since SVGs aren't automatically minified by Kibana when loaded like this, could you please minify this SVG manually (at least let's remove, the tons of inkscape, sodipodi tags and metadata tags in it), but ideally run it through something like SVGO, please.